File: sprint4/By languages/go/lesson10code1.go
func naive_solution(A int, x []int) []Triple {
    n := len(x)
    triples := make([]Triple, 0)
    for i1 := 0; i1 < n; i1++ {
        for i2 := i1 + 1; i2 < n; i2++ {
            for i3 := i2 + 1; i3 < n; i3++ {
                if x[i1]+x[i2]+x[i3] == A {
                    triples = append(triples, Triple{x[i1], x[i2], x[i3]})
                }
            }
        }
    }
    return triples
}

File: sprint4/By languages/python/lesson10code1.py
def naive_solution(A, x):
    n = len(x)
    triples = []
    for i1 in range(n):
        for i2 in range(i1 + 1, n):
            for i3 in range(i2 + 1, n):
                if x[i1] + x[i2] + x[i3] == A:
                    triples.append((x[i1], x[i2], x[i3]))
    return triples
